### ISSUE TITLE
loc works not properly for some cases.

### ISSUE THREAD

**Issue by itholic** [databricks/koalas#1173]
assuming that we have `Series` like the below

```python
>>> kser
7    10
6    20
5    30
4    40
3    50
2    60
1    70
Name: 0, dtype: int64
```

for given `kser`, below examples are working differently from pandas.
(`pser` is pandas' `Series` that has exactly same shape with `kser`)

```python
>>> pser.loc[6:4]
Series([], dtype: int64)

>>> kser.loc[6:4]
6    20
5    30
4    40
Name: 0, dtype: int64
```
```python
>>> pser.to_frame().loc[6:4]
    0
6  20
5  30
4  40

>>> kser.to_frame().loc[6:4]
Empty DataFrame
Columns: [0]
Index: []
```

**Comment by itholic** [comment]
i'm working on here

**Comment by itholic** [comment]
this is resolved at #1159 , but let it open for now, since that PR is not yet merged
